docs/kubernetes.md

```diff
@@ -45,7 +66,14 @@ inv knative.install
 
 # Check
 kubectl get pods -n knative-serving
-```  
+```
+
+If you are running a local cluster or one on AKS, you will have to update the `externalIPs`
+field in the `upload-service.yml`.
+You can query it with:
+```bash
+kubectl get svc istio-ingressgateway -n istio-system -o=jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
```

Comment:
It seems like we have two places for this configuration now, the `knative_route.sh` script and the README. Could you add this line to the `knative_route.sh` script and make it print out there?

If it's not in a script, a command like this tends to rot over time.